libraries/checkers/usfm_checker.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import print_function, unicode_literals
+import codecs
+import os
+import traceback
+from libraries.checkers.checker import Checker
+from libraries.door43_tools.page_metrics import PageMetrics
+from libraries.usfm_tools import verifyUSFM, usfm_verses
+
+
+class UsfmChecker(Checker):
+
+    def __init__(self, preconvert_dir, converted_dir, log=None):
```

Comment:
It's always best to not to specifically name variables used by the parent init, as those may change (which they have in my branch). Do this instead for the first two lines:
```
    def __init__(self, *args, **kwargs):
        super(UsfmChecker, self).__init__(*args, **kwargs)
```